### PR TITLE
Metadata pipeline

### PR DESCRIPTION
**Pull request by sdavrieux**
- Created metadata pipeline
- Added structure to the project
- Added settings and flags system
- Created object to interact easily with AzureML

### CODE REVIEW

File: blurring_as_a_service/metadata_pipeline/submit_metadata_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    ]
+
+    pipeline_job = aml_interface.submit_pipeline_job(
+        pipeline_job=metadata_pipeline_job, experiment_name="metadata_pipeline"
+    )
+
+    aml_interface.wait_until_job_completes(pipeline_job.name)
+
+
+if __name__ == "__main__":
```
Comment (epureanudiana) [inline blurring_as_a_service/metadata_pipeline/submit_metadata_pipeline.py:70]: is this the main for running the metadata pipeline?

Comment (sdavrieux) [reply blurring_as_a_service/metadata_pipeline/submit_metadata_pipeline.py:70]: Yess, indeed, this submits the pipeline and commands to AzureML to be run on the compute instance.

File: blurring_as_a_service/metadata_pipeline/utils/coco_to_yolo_converter.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
-def _update_category(annotation):
-    annotation["category_id"] = annotation["category_id"] - 1
-    return annotation
-
-
 class CocoToYoloConverter:
+    """
+    Converts a COCO annotation dataset to a YOLOv5 format.
+    - The bbox will remain the same format;
```
Comment (epureanudiana) [inline blurring_as_a_service/metadata_pipeline/utils/coco_to_yolo_converter.py:7]: The bbox changes from `x_min, y_min, width, height` to `x_center, y_center, width, height`.

**Review by epureanudiana — APPROVED** [review]
nice work, very structured and clean!
I left a few comments, but most was covered yesterday, so I think we can merge

**Pull request merged**